spec/requests/admins/questions_spec.rb

```diff
@@ -12,12 +12,13 @@
         sign_in admin
         post admin_study_survey_questions_path(study, survey),
              params: {
-               question: { content_attributes: { type: 'label', label_text: 'Question' } }
+               question: { content_attributes: { type: 'label', required: true, label_text: 'Question' } }
```

Comment:
Can you change the value of required to false so we can verify that the value changes through the controller work as expected?

Reply:
of course

Reply:
Done